test/e2e/storage/vsphere/vsphere_volume_vsan_policy.go

```diff
@@ -341,3 +339,22 @@ func invokeStaleDummyVMTestWithStoragePolicy(client clientset.Interface, masterN
 	framework.ExpectNoError(err)
 	framework.ExpectEqual(isVMPresentFlag, false, errorMsg)
 }
+
+func getControlPlaneNode(client clientset.Interface) (string, error) {
```

Comment:
with the prior PRs, we now seem to have this similar function in a number of places (but with different error messages, context i guess)
should it have been generalized and placed in a shared location?

Reply:
ok, SGTM.